cs/Serializer/VowpalWabbitDefaultMarshallerExt.tt

```diff
@@ -0,0 +1,194 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
I'm assuming the corresponding 2970 lines .cs file is automatically generated.  Right?  Can that be left out?